datafusion/physical-expr/src/aggregate/sum.rs

```diff
@@ -511,28 +374,6 @@ mod tests {
 
     #[test]
     fn sum_decimal() -> Result<()> {
-        // test sum
```

Comment:
Since we alter the summation operation and added the subtraction operation inside the `ScalarValue` implementation, we moved the relevant tests under the `scalar.rs`.